demos/general/demo_OnACID.py

```diff
@@ -18,6 +18,7 @@
 from caiman.source_extraction import cnmf as cnmf
 from caiman.paths import caiman_datadir
 
+%matplotlib qt #otherwise vis will not work
```

Comment:
We can't do this (see below)

Reply:
Fixed in `patched qt matplotlib ipython magic` commit below. Good call.